airtable/introduction/intro-airtable/intro-airtable.md

```diff
@@ -0,0 +1,34 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Airtable?
+
+---
+## Content
+
+Part database, part spreadsheet, Airtable was introduced in 2012 and has since continued to grow.
+
+Airtable can be used as a simple spreadsheet, but most of its power comes from the ability to introduce multiple data types in a cell (e.g. documents, images etc.), especially when compared to other spreadsheet applications (such as Excel or Google Sheets). You can say that Airtable was designed for data entry.
+
+Another advantage that Airtable has over traditional spreadsheets is that you can group rows by their values and link different data tables together.
+
+Airtable can be used as a production database for your application, a personal spreadsheet or more.
+
+---
+## Practice
+
+Airtable is part ??? and part ???.
```

Comment:
This question should be changed. If a user tries `spreadsheet` first and `database` 2nd, it would show as incorrect.

Reply:
agreed, didn't think about this when making the question.